junifer/testing/datagrabbers.py

```diff
@@ -141,8 +144,8 @@ def get_item(self, subject: str) -> Dict[str, Dict]:
         anat_fname = self.datadir / f"{subject}_T1w.nii.gz"
         nib.save(fmri_img, fmri_fname)
         nib.save(anat_img, anat_fname)
-        out["BOLD"] = {"path": fmri_fname}
-        out["T1w"] = {"path": anat_fname}
+        out["BOLD"] = {"path": fmri_fname, "space": "MNI"}
```

Comment:
I'd rather use the right space here, so we test correctly the masks/parecellations.